installer/scripts/omsadmin.sh

```diff
@@ -353,6 +383,17 @@ onboard_scom()
     #Always register SCOM as secondary Workspace
     echo "SCOM Workspace" > $CONF_DIR/.multihoming_marker
     configure_logrotate
+
+    #Open port 1270 if not already open
+    is_scom_port_open
+    IS_PORT_OPEN=$?
+    if [ $IS_PORT_OPEN -eq 1 ]; then
+        echo "Opening port 1270"
+        /opt/omi/bin/omiconfigeditor httpsport -a 1270 < /etc/opt/omi/conf/omiserver.conf > /etc/opt/omi/conf/omiserver.conf_temp
```

Comment:
I'm not 100% sure how `omiconfigeditor` works. Yeah, I know I wrote it, but it's been a while.

Is `omiconfigeditor` smart enough to uncomment a line if it's commented out when you add a parameter to a line? Worth checking, just to be 100% certain this will work.

My concern is something like this in `omiserver.conf`:

```
#httpsport=0
```

If you invoke `omiconfigeditor` with the `-a` qualifier, will it be smart enough to uncomment the line and then add the option?